src/js/pensions/config/form.js

```diff
@@ -199,10 +199,28 @@ const formConfig = {
               'ui:widget': 'yesNo'
             },
             severancePay: {
```

Comment:
Thanks @J-Quag, @jbalboni for noting the UI for this needed to be synced up with the schema.